packit_service/worker/github_handlers.py

```diff
@@ -412,10 +437,42 @@ def run(self) -> HandlerResults:
             git_project=self.project, working_dir=self.config.command_handler_work_dir
         )
 
-        pipeline_id = str(uuid.uuid4())
-        payload: dict = {"pipeline": pipeline_id}
+        chroots = self.job.metadata.get("targets")
+        for chroot in chroots:
+            pipeline_id = str(uuid.uuid4())
+            payload: dict = {
+                "pipeline": pipeline_id,
+                "token": self.config.testing_farm_secret,
+            }
+
+            stg = "-stg" if self.config.deployment == Deployment.stg else ""
+            copr_repo_name = (
+                f"{self.project.namespace}-{self.project.repo}-"
+                f"{self.pr_event.pr_id}{stg}"
+            )
 
-        # TODO change to real url
-        self.send_testing_farm_request("url", "POST", {}, payload)
+            payload["artifact"] = {
+                "repo-name": self.pr_event.base_repo_name,
+                "repo-namespace": self.pr_event.base_repo_namespace,
+                "copr-repo-name": copr_repo_name,
+                "copr-chroot": chroot,
+                "commit-sha": self.pr_event.commit_sha,
+                "git-url": self.pr_event.https_url,
+                "git-ref": self.pr_event.base_ref,
+            }
+
+            logger.debug("Sending testing farm request...")
+            logger.debug(payload)
+
+            req = self.send_testing_farm_request(
+                TESTING_FARM_TRIGGER_URL, "POST", {}, payload
+            )
+            if not req:
+                logger.debug("Failed to post request to testing farm API.")
+                return HandlerResults(success=False, details={})
```

Comment:
small suggestion: would be nice to post a comment to the PR so that we don't have to look in logs if something goes wrong

Reply:
Don't you like searching in logs? 😁J.K.

I agree that this could be in a comment.